src/pages/job-offer-reject/overview/components/offered/offered.services.ts

```diff
@@ -1,6 +1,6 @@
 import { isoToStandard } from '../../../../../core/time';
 import { Offer } from '../../../../../core/types';
```

Comment:
use api package  types instead of core type and use abs path

Reply:
This component is related to job offer not chats, I didn't change anything except a wrong path which caused build error